module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Enable-EntraAzureADAlias.md

```diff
@@ -9,25 +19,28 @@ schema: 2.0.0
 # Enable-EntraAzureADAlias
 
 ## Synopsis
-{{ Fill in the Synopsis }}
+
+Enables aliases for AzureAD modules.
 
 ## Syntax
 
-```
+```powershell
 Enable-EntraAzureADAlias
 ```
 
 ## Description
-{{ Fill in the Description }}
+
+Enables aliases for Azure AD modules.
 
 ## Examples
 
-### Example 1
+### Example 1: Enable aliasing
+
 ```powershell
-PS C:\> {{ Add example code here }}
+Enable-EntraAzureADAlias
 ```
 
-{{ Add example description here }}
+Enables all Azure AD prefixes for the current PowerShell session.
```

Comment:
Enables aliases with AzureAD prefix for the current PowerShell session.

Reply:
Good catch. Applied the recommendation.